openmc/tallies.py

```diff
@@ -897,10 +904,18 @@ def get_values(self, scores=[], filters=[], filter_bins=[],
 
                 # Add indices for each bin in this Filter to the list
                 for bin in bins:
-                    filter_indices[i].append(
-                        self.get_filter_index(filter.type, bin))
+                    if user_filter:
+                        filter_indices[i].append(
+                            self.get_filter_index(filter.type, bin))
+                    else:
+                        filter_indices[i].append(
+                            self.get_filter_index(filter.type, bin))
+
+                # Account for stride in each of the previous filters
+                reverse_mult = lambda x: [xi * filter.num_bins for xi in x]
```

Comment:
Sure, the lambda isn't needed now anyway since I simplified to your proposed version below.